cpu/mem scalers can scale to zero (#1088)
kedacore/keda-docs: content/docs/2.11/scalers/cpu.md

> **Notice:**
> - This scaler **requires prerequisites**. See the 'Prerequisites' section.

> > - This scaler can scale to 0 only when user defines atleast one external scaler (eg. Kafka + cpu/memory, or Prometheus + cpu/memory) and sets minReplicaCount to 0.

Reviewer: ```suggestion
> - This scaler can scale to 0 only when user defines at least one additional scaler which is not CPU or Memory (eg. Kafka + CPU/Memory, or Prometheus + CPU/Memory) and sets minReplicaCount to 0.
```

> > - This scaler can scale to 0 only when user defines at least one additional scaler which is not CPU or Memory (eg. Kafka + CPU/Memory, or Prometheus + CPU/Memory) and sets minReplicaCount to 0.

Reviewer 2: ```suggestion
> - This scaler can scale to 0 only when user defines at least one additional scaler which is not CPU or Memory (eg. Kafka + CPU, or Prometheus + CPU) and sets minReplicaCount to 0.
```

Author: ```suggestion
> - This scaler can scale to 0 only when user defines at least one additional scaler which is not CPU or Memory (eg. Kafka + CPU, or Prometheus + CPU) and minReplicaCount is 0.
```
what about not using the word `set` also, since minReplicaCount is by default zero therefore it doesnt have to be `set` necessarily?

Reviewer 2: Makes sense. Also annotate rename minReplicaCount to `minReplicaCount` please
